app/controllers/admin/enterprise_fees_controller.rb

```diff
@@ -2,11 +2,13 @@
 
 require 'open_food_network/order_cycle_permissions'
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
We should add the filename to the rubocop todo file instead of disabling the cop here.